bindings/python/pymongoarrow/schema.py

```diff
@@ -86,3 +86,22 @@ def __eq__(self, other):
         if isinstance(other, type(self)):
             return self.typemap == other.typemap
         return False
+
+    @classmethod
+    def from_arrow(cls, aschema: pa.Schema):
```

Comment:
Should we make the constructor accept `pa.Schema` rather than introduce a new api method?

Reply:
Same same but different. This was quick. I time-boxed my work, and stopped myself short of  bigger refactoring. I would love to discuss.

Reply:
The difference is that we try to avoid adding new public methods unless there's a clear need.

Reply:
We created https://jira.mongodb.org/browse/ARROW-220 after discussing further.